pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/BasicModelDefTest.java

```diff
@@ -1260,6 +1260,22 @@ public void parallelStagesFailFast() throws Exception {
                 .go();
     }
 
+    @Issue("JENKINS-53558")
+    @Test
+    public void parallelStagesFailFastWithOption() throws Exception {
+        expect(Result.ABORTED,"parallelStagesFailFastWithOption")
+                .logContains("[Pipeline] { (foo)",
+                        "[first] { (Branch: first)",
+                        "[Pipeline] [first] { (first)",
+                        "[second] { (Branch: second)",
+                        "[Pipeline] [second] { (second)",
+                        "SECOND STAGE ABORTED")
+                .hasFailureCase()
```

Comment:
Maybe add `.logNotContains("Second branch")`, just to make sure we get killed during the `sleep`.